src/peergos/server/tests/linux/S3UserTests.java

```diff
@@ -29,6 +29,7 @@ public class S3UserTests extends UserTests {
 
     private static Args pkiArgs = buildArgs()
             .with("useIPFS", "true")
+            .with("transactions-sql-file", "transactions.sql")
```

Comment:
This should have a default value no? Then we don't need to set it to the same value everywhere?